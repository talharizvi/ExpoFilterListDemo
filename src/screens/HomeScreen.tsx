import React, { useState, useEffect} from 'react';
import { StatusBar } from 'expo-status-bar';
import { StyleSheet, Text, View } from 'react-native';
import colors from '../constants/colors';


const HomeScreen:React.FC=()=> {
    return (
      <View style={styles.container}>
        <Text>Empty Screen</Text>
      </View>
    );
}

const styles = StyleSheet.create({
    container: {
      flex: 1,
      backgroundColor: colors.primary,
      alignItems: 'center',
      justifyContent: 'center',
    },
});  

export default HomeScreen
