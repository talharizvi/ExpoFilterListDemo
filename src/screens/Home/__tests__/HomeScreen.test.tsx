import React from 'react';
import HomeScreen from '../HomeScreen';
import { render, screen } from '@testing-library/react-native';

describe('HomeScreen', () => {
    it('renders correctly', () => {
      render(<HomeScreen/>)
      expect(screen.getByText('Empty Screen')).toBeDefined()
    });
  });