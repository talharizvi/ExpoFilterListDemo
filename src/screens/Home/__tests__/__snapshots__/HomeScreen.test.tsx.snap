// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Empty Screen
  </Text>
</View>
`;
